sample-project/deploy/1_tokens.migration.js

```diff
@@ -32,19 +33,23 @@ module.exports = async (deployer, logger, verifier) => {
     []
   );
 
-  // TODO: finalize message
   /*
-   * Under the hood the `deployer.deployProxy(...)` function will call `linkProxyToImplementation`
+   * Under the hood the `deployer.deployProxy(...)` function will call `verifyProxy`
    * function, to automatically verify that this contract is a proxy on etherscan.
-   * If you deploy the contract with this function on blockscout, you will get the following message:
-   * -- Possibly you are trying to verify a contract on a blockscout! --
+   * If you deploy the contract with this function on BlockScout, you will get the following message:
+   * -- Perhaps you are trying to verify a contract on the BlockScout. Proxy verification failed! --
```

Comment:
```suggestion
   * -- Perhaps you are trying to verify a contract on BlockScout. Proxy verification failed! --
```

BlockScout is a name of a thing (forgot the term :)) no preposition is needed